opendap/src/main/java/opendap/servlet/AbstractServlet.java

```diff
@@ -413,19 +413,22 @@ public void IOExceptionHandler(IOException e,  ReqState rs) {
    * @param e        The exception that caused the problem.
    * @param rs The <code>ReqState</code> for the client.
    */
-  public void anyExceptionHandler(Throwable e, ReqState rs) {
-    HttpServletResponse response = rs.getResponse();
+  public void anyExceptionHandler(Throwable e, ReqState rs)
+  {
+    log.error("DODServlet ERROR (anyExceptionHandler): " + e);
+    e.printStackTrace();
+    printThrowable(e);
```

Comment:
Now that I think about it, I'm not completely sure that that method will print the exception's stack trace to the log. Maybe 95% sure. Wouldn't hurt to test.